modules/claims_api/app/swagger/claims_api/v0/form_526_controller_swagger.rb

```diff
@@ -93,8 +93,8 @@ class Form526ControllerSwagger
               **Flashes and special issues**<br/>
```

Comment:
Not gonna hurt anything, but no need to update this since V0 is deprecated.
We have a ticket to begin removing all the V0 code, but we just haven't got to it yet.